### DISCUSSION TITLE
Disable a certain script?

### DISCUSSION THREAD

**Discussion by quadcom** [RandomNinjaAtk/docker-sonarr-extended#13] [Q&A] [answered]
Is it possible to disable the QueueCleaner.bash script?

My workflow requires that I sync DLs from the remote seedbox to the SOnarr instance. This takes time, and if it does not happen fast enough, Sonarr times out, and I believe the script is flagging "waiting to import" times as manually failed.

It would be great if there were a script that would allow remote servers to sync DLs and then have Sonarr process the downloads after the sync is done. So far, I have found nothing that would solve this problem.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Set the parameters correctly.... (Hint: enableQueueCleaner) ...
https://github.com/RandomNinjaAtk/docker-sonarr-extended#parameters
